api/db/build.gradle

```diff
@@ -0,0 +1,21 @@
+buildscript {
+    repositories {
+        mavenCentral()
+    }
+    dependencies {
+        classpath 'org.liquibase:liquibase-gradle-plugin:1.2.4'
+        classpath 'mysql:mysql-connector-java:5.1.37'
+    }
+}
+apply plugin: 'org.liquibase.gradle'
+
+liquibase {
+    activities {
+        main {
+            changeLogFile 'changelog/db.changelog-master.xml'
+            url 'jdbc:mysql://localhost:3306/workbench'
```

Comment:
Is there an easy way to parameterize host, port, username, and password (e.g. from environment vars + defaults as specified here)? This might help a bit in the CI environment, which we'll need soon...and we'll also need to pass in different values to do updates in GCP environments eventually. Happy to wait if you prefer